src/renderer/uia/UiaRenderer.cpp

```diff
@@ -274,106 +267,6 @@ CATCH_RETURN();
     return S_FALSE;
 }
 
-// Routine Description:

```

Comment:
Boi do I love deleting empty implementations like these. It's satisfying for some reason. Everyone should give it a try 😄